syft/generic/pointers/pointer_plan.py

```diff
@@ -93,6 +93,26 @@ def __call__(self, *args, **kwargs):
 
         return response
 
+    def parameters(self) -> List:
+        """Return a list of pointers to the plan parameters"""
+
+        assert (
+            len(self._locations) == 1
+        ), ".parameters() for PointerPlan with > 1 locations is currently not implemented."
+        # TODO implement this feature using MultiPointerTensor
```

Comment:
Q: I have an idea, but I do not know if it is a good one or not. What do you think of linking issues directly in the code? (like create the issue and put the issue number as comment)

Reply:
Definitely a good idea :)

Reply:
I don‘t think it is a good idea to link github issues in code, this should be separated. Not every part could be matched to one issue over time. And imagine e.g. moving to gitlab ;-).

Reply:
Yeah, I don't think every part could be linked to an issue (that's what the history and `git blame` are for anyway), but it might be helpful in `TODO` comments to be able to link to an issue with further info and discussion so that stuff doesn't have to live in the comments. I have no strong opinions here, just wanted to chime in to say that I can see potential value. 🤷‍♂

Reply:
I 100% support TODO: <link to issue>